optimade/validator/utils.py

```diff
@@ -214,29 +223,39 @@ def get(self, request: str):
 
         status_code = None
         retries = 0
+        errors = []
         # probably a smarter way to do this with requests, but their documentation 404's...
```

Comment:
Does it still 404?

Reply:
I can't even find the link anymore, the comment can probably be removed as I think the implicit retries that the `requests` library can do are not suitable here anyway

Reply:
```suggestion
```